src/mpyl/cli/plan.py

```diff
@@ -61,16 +62,25 @@ def plan(ctx, config, properties):
     )
 
 
-@plan.command("create")
+@plan.command("discover")
+@click.option(
+    "--project",
+    "-p",
+    type=click.STRING,
+    required=False,
+    help="Limit the run plan to only this project",
+)
 @click.pass_obj
-def create_plan(ctx: Context):
+def discover_plan(ctx: Context, project):
```

Comment:
Is there no typing available for `project`?

Reply:
Most other usages [do not](https://github.com/Vandebron/gh-mpyl/blob/main/src/mpyl/cli/build.py#L69) [define](https://github.com/Vandebron/gh-mpyl/blob/main/src/mpyl/cli/build.py#L107-L111) [types](https://github.com/Vandebron/gh-mpyl/blob/main/src/mpyl/cli/projects.py#L53) so I assumed this was not available, but now that you asked I actually did find [one](https://github.com/Vandebron/gh-mpyl/blob/main/src/mpyl/cli/projects.py#L166) that does so I guess it is available?

I'll raise a follow-up PR to do this 👍